docker-compose.yaml

```diff
@@ -4,23 +4,17 @@ services:
 
   caddy:
     container_name: caddy
-    image: abiosoft/caddy:1.0.3-no-stats
-    ports:
-      - 80:80
-      - 443:443
+    image: caddy/caddy:2-alpine
     network_mode: host
-    command: -email ${LETSENCRYPT_EMAIL} -agree=${LETSENCRYPT_AGREE} -log stdout -host ${SEARX_HOSTNAME} -conf /etc/Caddyfile
+    command: caddy run -config /etc/caddy/Caddyfile
     volumes:
-      - ./Caddyfile:/etc/Caddyfile:rw
-      - ./caddy:/root/.caddy:rw
-      - ./srv:/srv:rw
+      - ./Caddyfile:/etc/caddy/Caddyfile:rw
```

Comment:
could be set to `./Caddyfile:/etc/caddy/Caddyfile:ro` because the config doesn't change

Reply:
Updated.